### ISSUE TITLE
SPLIT & COPY  Fails On Concept Label - Split To - 3.8.0qa2 

### ISSUE THREAD

**Issue by singhik** [NCIEVS/nci-protege5#529]
@bdionne : Split Complex Edit is failing to keep 'split to' concept name, 'Split to' concept gets renamed to 'split from' concept , below are the screen prints , NCI Edit Tab v1.3.3. : 

![image](https://user-images.githubusercontent.com/36314371/130703032-d4dbfd67-c76c-45c6-a0f9-34ac9821eb09.png)

![image](https://user-images.githubusercontent.com/36314371/130703051-efa80281-f74f-4959-82f9-32ebe8a99bb5.png)

![image](https://user-images.githubusercontent.com/36314371/130703068-46687d84-f20c-47ed-b180-077b1f7ad514.png)

![image](https://user-images.githubusercontent.com/36314371/130703076-9a1b83b9-a087-42cd-9dc2-10436a10d11f.png)

![image](https://user-images.githubusercontent.com/36314371/130703089-3aef9fdb-582e-403d-b87e-dd2ed17ad6b0.png)

![image](https://user-images.githubusercontent.com/36314371/130703111-78357574-110b-44a0-947d-9063af1c3300.png)

**Comment by bdionne** [comment]
This bug appears in v1.3.2 also

**Comment by singhik** [comment]
@bdionne : COPY, also fails to keep label of 'Copy To' Concept,

**Comment by singhik** [comment]
Comments from @bdionne  : 

""We discussed this , this relates to the business rules. In your test case where you have the complex property value set to the name of the class, that’s ok except for the FULL_SYN property. Every class has a FULL_SYN prop whose value is equal to the rdfs:label and also the Preferred_Name property. If we allowed copying that property, when you save you see the warning that there can only be one FULL_SYN. In the case of split if you then delete the wrong one you end up with duplicate names. 

So we don’t copy FULL_SYNs whose value is equal to the class name. It’s actually not only the rdfs:label, it whatever property is set in protege properties.

We do allow copying of other properties, .eg. DEFINITION, and those might also trigger warnings, .eg. there can only be one with NCI source, but in this cases the modeler can edit them and they still might be useful.""
